### PR TITLE
Replaces head of staff telebaton with energy variant - Special snowflake edition

### PR DESCRIPTION
**Pull request by silicons**
Replaces head of staff telebaton with an energy variant that self-recharges and consumes charge.
It holds enough charge to HIT 15 TIMES.
However:
Each hit only does a MAXIMUM of Weaken(1) - It does not stack.
Hitting a target consecutively only gives you a clickdelay of 3, which is a bit faster than normal melee attacks.
After 5 hits, the target is stunned for 5 seconds, and you get a delay of about 2 seconds until you can hit again.
Breaking the consecutive target chain will incur a 2 second delay before you can stun anyone again.
Hitting a target will break your pull on that target, if you are pulling them.
The baton can NOT be used if it is out of charge, and it recharges 1 hit's worth of charge every 2 seconds (depending on lag, really.)
If you're on harm intent, each hit will do 2 brute.

**tl;dr it can be used to reliably throw someone out of your department assuming you don't get disarmed but it now sucks absolute shit in anything else because it has extremely limited charges and you MUST stay on a target for about 1.5 seconds assuming you can click 3-4 times a second and if you get attacked by anything more than 1 person you're royally fucked even harder than you were with telebatons before this. Also, dragging someone to the brig will be hard when you can't pull with it, and if you say cuffing is still a thing, well, it was always a thing with any stun weapon that stuns for 2 Life() ticks or longer.**
codersprites
telebaton_extended

### CODE REVIEW

**Comment by Armhulen** [comment]
pictures needed

**Comment by dannno** [comment]
Variant*

**Comment by Repukan** [comment]
Sounds about as good as the pAi's

**Comment by kevinz000** [comment]
@dannno appreciated :P

**Comment by kevinz000** [comment]
@Repukan they wanted a nerf this is a nerf or do you mean it makes them powerful?

**Comment by kevinz000** [comment]
@Armhulen literally the stock picture but it glows. but fine i'll get them shortly.

**Comment by Repukan** [comment]
Too weak and overly complicated.

**Comment by Repukan** [comment]
And they didn't want a nerf, they wanted it removed and flashes buffed

**Comment by kevinz000** [comment]
@Repukan great well i don't want it removed so this is a nerf to make them emergency self-defense weapons again.

**Comment by Repukan** [comment]
Yeah and now they suck at being a self defense weapon.

Why even use such a clunky POS when I can just scrunch up a stunprod and tase the nerds.

**Comment by kevinz000** [comment]
@Repukan because stunprods fit in pockets and heads of staff start out with a taser?

**Comment by Repukan** [comment]
What taser?

Heads don't start with tasers, at most I can imagine the gun from the HoP. Functionally this is not even close to a taser because give hits I'd too long to actually defend yourself. You really think you'll last to get the five hits in before someone disarms you or a dual esword cuts you into ribbons?

I know a buff or nerf is needed but seriously this just seems clunky on paper and it sounds like hell on gameplay.

**Comment by kevinz000** [comment]
... I was being sarcastic. Stunprods do not fit in backpacks and heads of staff except for HOS do not start out with tasers (hos gets armory so he doesn't get a baton anyways)
Also @Repukan the first hit weakens.

**Comment by Repukan** [comment]
I'd recommend adding a piece of paper that explains how it works at least.

Head players are usually fucking clueless, which is what worries me.

**Comment by kevinz000** [comment]
@Repukan good point on that.

**Comment by optimumtact** [comment]
You're nerfing something down to the point where it may as well not exist, just finish the job

**Comment by kevinz000** [comment]
@optimumtact It's still useful, just less useful for beating antags up with and more useful for actually defending youreself.

**Comment by Iamgoofball** [comment]
Just remove it entirely you autist

**Comment by optimumtact** [comment]
just let it happen bro shhhhhh

**Comment by kevinz000** [comment]
>not hanging on to my favorite head of staff item

**Comment by Armhulen** [comment]
>defending yourself

>2 brute

lol just remove it

**Comment by Durkel** [comment]
Replace them with sick sticks from minority report, they make you vomit on hit. I have no idea on how to balance it but I want to make people hurl.

**Comment by kevinz000** [comment]
@Armhulen it's the stun, not the 2 brute, also 10 brute over 1.5 seconds is not that much worse if you can stun at the same time. It's meant to stun, why are we focusing on its low brute damage on harm intent?

**Comment by ExcessiveUseOfCobblestone** [comment]
> a melee stun is as good as the ranged one

yes that is the goal

**Comment by KorPhaeron** [comment]
>16 new vars

**Comment by Arianya** [comment]
>5 seconds
>Long enough to throw someone out

Doesn't every airlock take like a full second to run through its open animation?

Engineering and Science both have double sets of airlocks that will add atleast 2-3 seconds to throw someone out, leaving you only 2 seconds to pull them to the airlocks, get them out, and close atleast the exterior airlock behind you as you re-enter the department.

Instead of nerfing it in to near oblivion, why not just take the existing telebaton and give it charges, which solves the infinistun issue?

Hell you could even keep the existing icon and just fluff it as the baton breaking down after repeated use and so you need to repair it with a welder, which has a usetime and requires eye protection (admittedly the latter isn't very difficult for most heads, but still)

Comparing it to the stats of this item, say a 10 second usetime to restore one "charge", max charges of 3-4?

**Comment by kevinz000** [comment]
*shrugs*
